django/template/backends/base.py

```diff
@@ -73,9 +75,7 @@ def iter_template_filenames(self, template_name):
         directory traversal attacks.
         """
         for template_dir in self.template_dirs:
-            try:
-                yield safe_join(template_dir, template_name)
-            except SuspiciousFileOperation:
+            with suppress(SuspiciousFileOperation):
                 # The joined path was located outside of this template_dir
```

Comment:
There are still a few cases like this where the comment's connection to `suppress()` is less obvious than it was when try/except was used. You could move this above the with and rephrase it like "SuspiciousFileOperation occurs if the jointed path is located outside...."